Chat/src/app/utils/getToken.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export type UserToken = {
+  expiresIn: number;
+  identity: string;
+  token: string;
+};
+
+// TODO: Need to revisit if cachedUserToken with respect to auto refresh token
```

Comment:
We should resolve this TODO

Reply:
Created ADO item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691382